### ISSUE TITLE
Useing in angular 2+

### ISSUE THREAD

**Issue by moshe-l** [gavinmcfarland/flex-gap-polyfill#27]
How this polyfill can used in angular 2 application?
How can i pass my css to the proccess function in polyfill.ts file? 
Thanks a lot.

**Comment by limitlessloop** [comment]
Hi @moshe-l . The polyfill requires PostCSS. I've not used Angular before but from what I've read you need to use a custom Webpack config to use PostCSS. I found this article which talks about using Tailwind and angular, you should be ale to follow the same steps (without the Tailwind stuff).

https://dev.to/seankerwin/angular-8-tailwind-css-guide-3m45

**Comment by moshe-l** [comment]
Thanks you for your response. I'll check it out.

**Comment by limitlessloop** [comment]
I tried to have a go but I wasn't familiar enough with Angular. Let us know if you get it working, or if you hit a roadblock.

**Comment by madmacc** [comment]
@moshe-l Did you get this working with Angular?
Any tips?

**Comment by gavinmcfarland** [comment]
I found the following articles talking about enabling PostCSS support for Angular and it seems like they've ditched it for now. It's not been supported for a few versions now. Somebody talks about getting it to work with a custom build. It may be useful for someone that's interested in trying it.

Various links I found below about it below:

https://github.com/angular/angular-cli/issues/26536
https://github.com/nrwl/nx/issues/9650
https://medium.com/@kabya78890/install-tailwind-css-using-post-css-in-your-angular-project-9cc0f9a3927b

I'll be closing this issue for now.
